packages/subgraph/schema.graphql

```diff
@@ -79,6 +79,28 @@ type RegisterNetworksAndAliasesMessage implements Message @entity {
   addCount: BigInt!
 }
 
+type CorrectLastEpochMessage implements Message @entity {
+  id: ID!
+  block: MessageBlock!
+  data: Bytes
+  newMerkleRoot: Bytes!
+  corrections: [LastEpochCorrection!]! @derivedFrom(field: "message")
+}
+
+type LastEpochCorrection @entity {
```

Comment:
This is not needed as a separate entity now, we can put the data from here into CorrectLastEpochMessage